pkg/registry/batch/job/strategy_test.go

```diff
@@ -892,6 +899,32 @@ func TestJobStrategy_WarningsOnCreate(t *testing.T) {
 				Spec: validSpec,
 			},
 		},
+		"high completions and parallelism": {
```

Comment:
I would like to have an analogous test for updates to ensure both entry paths to the new code are adjusted.